### ISSUE TITLE
Only send alerts for single IFO triggers that both pass alert threshold and are found in coincidence with grb

### ISSUE THREAD

**Issue by cmessick** [mina19/approval_processorMP#2]
I was asked to open a PR about this so there's an official record somewhere. This is just a request to modify AP to only label single IFO gracedb uploads with ADVREQ if they 

1. Pass the alert FAR threshold and
2. are found in coincidence with a GRB

It looks like Raven already updates event logs with information about possible coincidences (see e.g. https://gracedb.ligo.org/events/view/T276080, under "External Coincidence"), so I believe the only change needed on the approval processor side will be differentiating between single and double IFO candidates, looking for this information from raven if a single ifo candidate, and then a conditional that describes when to apply the advreq label.

**Comment by pshawhan** [comment]
There is not a trivial solution since the external trigger may become known to RAVEN *after* the GW trigger has been passed to Approval Processor, and Approval Processor is designed to decide whether or not to process a GW trigger when it first learns about it.

**Comment by chadhanna** [comment]
Hi,

Could we consider simply having a patch to approval processor ignore single ifo results for now and then if someone needs to intervene manually they can do so?  At least then we can upload to gracedb.

thanks,

Chad

**Comment by pshawhan** [comment]
Yes, that is one approach, and in fact Deep has written code to do that.  I
assume that needs a quick review and then can be deployed.

Peter

**Comment by chadhanna** [comment]
Ok.  Great.  Can we move forward on this?  Looks like we are about to have
a stretch of llo only time.

**Comment by pshawhan** [comment]
We will have to make a case to Patrick -- will one of you talk to him?  The
policy up to now has been that even minor changes to the Approval Processor
code have been in the queue behind the major changes that Mina has been
working on.  But I would think he would be agreeable to deploying this
simple change this week.

Peter

**Comment by chadhanna** [comment]
I will ping him

**Comment by prbrady** [comment]
Deep and I talked yesterday about the status of this. The initial approach will not work.  He is looking at a different, and probably more sustainable approach, which we will go over this afternoon and hopefully discuss a patch tomorrow morning.
